features/spicy-protocol-analyzer/scripts/main.zeek@ALT-two-units@

```diff
@@ -6,6 +6,12 @@ export {
 	## Log stream identifier.
 	redef enum Log::ID += { LOG };
 
+	## The ports to register @ANALYZER@ for.
+	const ports = {
+		# TODO: Replace with actual port(s).
+		12345/@PROTOCOL_LOWER@ # adapt port number in @ANALYZER_LOWER@.evt accordingly
```

Comment:
Let's use the same string as in [features/spicy-protocol-analyzer/scripts/main.zeek@ALT-one-unit@](https://github.com/zeek/package-template/pull/28/files#diff-30cb6aaa39a8925279150faaa18600665d10c8b7e11b4678909939080d403c5e),

```suggestion
		# TODO: Replace with actual port(s).
		12345/@PROTOCOL_LOWER@,
```